src/server.js

```diff
@@ -139,22 +140,26 @@ const plugin =
                 });
               }
             } catch (e) {
-              const error = Object.getOwnPropertyNames(e).reduce(
-                (obj: any, key) => {
-                  obj[key] = e[key];
-                  return obj;
-                },
-                {}
-              );
-              delete (error: any).stack;
+              const error =
+                e instanceof ResponseError
+                  ? e
+                  : new Error(
+                      'UnknownError - Use ResponseError for more detailed error messages'
```

Comment:
If a client were to (incorrectly) just use the error message straight from the server, let it bubble, and render the error message on the client, then this developer error would be the one to surface right? Maybe it's best to leave out the `Use ResponseError for more detailed error messages` part?

Reply:
I think it is good for the developer experience, and doesn't really harm production